src/components/contextual/pages/pool/PoolMigrationCard/PoolMigrationCard.vue

```diff
@@ -0,0 +1,119 @@
+<script setup lang="ts">
+import useNetwork from '@/composables/useNetwork';
+import { deprecatedDetails } from '@/composables/usePool';
+import NewPoolData from './NewPoolData.vue';
+import usePoolsQuery from '@/composables/queries/usePoolsQuery';
+import usePoolQuery from '@/composables/queries/usePoolQuery';
+
+/**
+ * TYPES
+ */
+type Props = {
+  poolId: string;
+};
+
+/**
+ * PROPS
+ */
+const props = defineProps<Props>();
+
+/**
+ * COMPOSABLES
+ */
+const { networkSlug } = useNetwork();
+
+const poolDeprecatedDetails = computed(() => deprecatedDetails(props.poolId));
+
+let newPoolQuery;
+if (poolDeprecatedDetails.value?.newPool) {
+  newPoolQuery = usePoolQuery(poolDeprecatedDetails.value?.newPool);
+}
+
+let suggestedPoolsQuery;
+if (poolDeprecatedDetails.value?.suggestedPools?.length) {
+  suggestedPoolsQuery = usePoolsQuery(
+    ref([]),
+    {},
+    {
+      poolIds: ref(poolDeprecatedDetails.value.suggestedPools),
+      first: 1000,
+    }
+  );
+}
+
+/**
+ * COMPUTED
+ */
+const loadingNewPool = computed(
+  () => newPoolQuery && newPoolQuery.isLoading.value
+);
+const newPool = computed(() => {
+  if (!newPoolQuery) {
+    return;
+  }
+  return newPoolQuery.data.value;
+});
+
+const loadingSuggestedPools = computed(
+  () => suggestedPoolsQuery && suggestedPoolsQuery.isLoading.value
+);
+const suggestedPools = computed(() => {
+  if (!suggestedPoolsQuery) {
+    return;
+  }
+  return suggestedPoolsQuery.data.value?.pages?.[0].pools;
+});
+</script>
+
+<template>
+  <BalLoadingBlock
+    v-if="loadingSuggestedPools || loadingNewPool"
+    class="mb-4 h-60 pool-actions-card"
+  />
+  <BalStack v-else vertical>
+    <BalCard shadow="2xl" noPad class="rounded-xl migration-card" growContent>
```

Comment:
It's very possible if we start using this style of card we will need it again somewhere else, so nesting it withing this contextual directory is not ideal. Please create a component called `/components/cards/GradientCard.vue`. This will be an extension of BalCard that only handles setting a different style. Then use it in this component like:
```suggestion
    <GradientCard shadow="2xl" noPad growContent>
```

Reply:
done